lib/internal/Magento/Framework/Cache/Config/Reader.php

```diff
@@ -24,7 +30,7 @@ class Reader extends \Magento\Framework\Config\Reader\Filesystem
      * @param string $domDocumentClass
      * @param string $defaultScope
      */
-    public function __construct(
+    public function __construct( //phpcs:ignore Generic.CodeAnalysis.UselessOverridingMethod
```

Comment:
Move that to DocBlock above.

Reply:
Hello @lbajsarowicz 
Thank you for your remark.
I have fixed it. Please check.
Thank you in advance.